examples/README.md

```diff
@@ -0,0 +1,9 @@
+### Basic Odds-feed example
+
+##### How to run it
+ - Use Java 8
+ - ```../library/gradlew clean install -p ../library```
+ - ```./gradlew clean build```
+ - Set token in src/main/java/com/oddin/oddsfeed/examples.Main on line 29
```

Comment:
Nit:
```suggestion
 - Set token in src/main/java/com/oddin/oddsfeed/examples/Main.java on line 29
```

Reply:
Also! maybe read the token from env variable instead? This would be much easier to handle